data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -30,6 +34,14 @@ class TestETL:
     _TRANSFORM_CSV_FILE_NAME = "transform.csv"
     _OUTPUT_CSV_FILE_NAME = "output.csv"
 
+    # This *does* need to be updated in the child class. It specifies where the "sample data" is
+    # so that we do not have to manually copy the "sample data" when we run the tests.
+    _SAMPLE_DATA_PATH = (
+        pathlib.Path(__file__).parents[3] / "data" / "tmp" / "ExampleETL"
```

Comment:
We could also just make a unique snapshot directory per test that is within the same directory as the test file?

Reply:
Sure. I did not change the overall structure of the tests from the homegrown snapshot tests. I don't have strong feelings here and updated this -- but if you have more specific thoughts of where you'd like these to be saved, please let me know :)